builtin/rebase--helper.c

```diff
@@ -50,6 +52,10 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 		OPT_CMDMODE(0, "edit-todo", &command,
 			    N_("edit the todo list during an interactive rebase"),
 			    EDIT_TODO),
+		OPT_CMDMODE(0, "setup-reflog", &command,
+			    N_("setup the reflog action"), SETUP_REFLOG),
+		OPT_STRING(0, "commit-id", &commit_id, NULL,
+			   N_("the commit to checkout")),
```

Comment:
I would say that if setup-reflog should always be passed a commit id, then we may assume that the commit id always follow `--setup-reflog` and we don't need a special `--commit-id` option. See how it works for `--add-exec-commands`.

Reply:
Oh, it also looks like --checkout-onto always takes $orig_head, $onto, and $onto_name… so I will do something like that here, too. Sorry for the noise.